skeletons/et-utils-package-skeleton/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "TODO: @nzz/et-utils-<package-name>",
```

Comment:
Is it possible to directly set the package name to the string, I give as a parameter with the CLI command?